File: Ejercicio 3/Ejercicio 3 (Busqueda lineal).py
arreglo = [12, 23, 57, 45, 34, 67, 89, 98]
valorBuscado = 34

indices = []
for index, elemento in enumerate(arreglo):
    if elemento == valorBuscado:
        indices.append(index)

if len(indices) > 0:
    print("Valor encontrado en los índices:", ", ".join(map(str, indices)))
else:
    print("Valor no encontrado en el arreglo.")

File: Ejercicio 3/Ejercicio 3 (Busqueda lineal).cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        int[] arreglo = {12, 23, 57, 45, 34, 67, 89, 98};
        int valorBuscado = 23;
        
        List<int> indices = new List<int>();
        
        for (int index = 0; index < arreglo.Length; index++)
        {
            if (arreglo[index] == valorBuscado)
            {
                indices.Add(index);
            }
        }
        
        if (indices.Count > 0)
        {
            Console.WriteLine("Valor encontrado en los índices: " + string.Join(", ", indices));
        }
        else
        {
            Console.WriteLine("Valor no encontrado en el arreglo.");
        }
    }
}